docs/reference/team-processes/triage-process.md

```diff
@@ -4,108 +4,130 @@ title: Triage Owner Duties
 
-You’ll then look at each issue — though if you’re on the FxA team and the issue is one affecting SubPlat (or vice-versa), leave it as-is since it’ll be triaged in the associated triage meeting  — and do the following:
+When reviewing issues, leave tickets that are not applicable to your team as-is since those tickets will be triaged in the associated triage meeting. The following should be reviewed for each ticket:
 
-* **Check that the issue is clear and ready to be worked on.** If more info or context is needed that an engineer in the triage meeting can provide, either leave a summary comment of what’s stated (if it’s quick) or ask them to comment on it, and otherwise, add the appropriate “needs” label (e.g. `needsdiscussion`, `needslegal`, `needsproduct`, `needsux`) and/or @ mention someone who may be able to answer pending questions. If more info is needed and the issue seems high priority, consider pinging someone directly about the issue after triage completes.
-* **Make sure the issue type is correct.** The default when filing in Jira or from GitHub is type “task” but it might need to be changed to “spike” if it’s a research task or work breakdown task, or “bug” in which case you will want to fill out the “Found in” (environment) and “[Severity][bug-severity]” (S1 is highest, S4 is lowest) fields.
-* **Add an initial estimated number of story points** if someone on the call can [estimate][point-estimation].
-* **If applicable, add the appropriate epic, labels, and/or issue links.** It’s uncommon for a new issue in triage to lack an epic if it should be associated with one, but it’s good to be aware of what epics exist and what the roadmap looks like. Some common labels to consider using are `maintenance` and  `qa-`, but be sure to check out [labels we use][labels-we-use] to be aware of others. Issue links create a link between issues and can be “blocked by X issue,” “relates to X issue,” and more.
-* **Move the issue out of the backlog section.** If the issue seems high priority or someone on the call asks to be assigned, it might belong in the current or following sprint (update the “sprint” field if so), or at the top of the “to do” list. Either way, drag and drop the task up and out of the backlog section (or manually change the status and refresh) and the issue status will change from “new” to “to do”.
+- **Make sure the issue is clear and ready to be worked on.** If more info or context is needed that an engineer in the triage meeting can provide, either leave a summary comment of what’s stated (if it’s quick) or ask them to comment on it. You can also @-mention those who may be able to answer pending questions. If more info is needed and the issue seems high priority, consider pinging the reporter directly about the issue after the triage meeting.
```

Comment:
Nit: add comma after "context is needed"

Q: for my own self, what do you mean by "quick"?
Q: how do we determine "high priority" (can we link to where priorities are broken down)?

Reply:
> Q: for my own self, what do you mean by "quick"?
I think if you're able to gather the extra context to leave in a comment, then do so. If not, ask them to leave a comment.

> Q: how do we determine "high priority" (can we link to where priorities are broken down)?
It will be mentioned during the triage meeting.

Tagging @LZoog , as she is the initial author of this section, in case I didn't interpret it correctly.